docs/components/ComponentsList.tsx

```diff
@@ -1,8 +1,9 @@
 import * as React from 'react';
-import { ActionLink, Expander, styled, distance, Icon, ActionLinkProps, themed, reStyled } from '../../src';
+import { ActionLink, Expander, styled, distance, Icon, ActionLinkProps } from '../../src';
 
 // @ts-ignore
 import getUrl from 'react-styleguidist/lib/utils/getUrl';
+import { themed } from '../../src/utils/styled';
```

Comment:
Sure that themed is no no exported? I mean Zeisslets will / may depend on it. Same for reStyled.